### ISSUE TITLE
[ BUG ] Doesnt fire the TrackEndEvent and TrackStartEvent

### ISSUE THREAD

**Issue by BlyZeDev** [Yucked/Victoria#119]
<p align="center">
<img src="https://media.giphy.com/media/20JY76TfKAhR20SfJu/source.gif" width="25%"/>
</p>

---

### `Describe the bug/issue.`

Doesnt fire the TrackEndEvent and TrackStartEvent on this soundcloud track. ( https://soundcloud.com/thegreatnepyrus/mario-and-luigi-bowsers-inside-story-music-ost-04-boss-battle ).
This for example ( https://soundcloud.com/kifbenderclassical/mario-and-luigi ) works fine.
With youtube, youtube music or twitch it works perfectly fine.

### `Stacktrace / Screenshots`

*When debugging yourself, you can attach screenshots of Autos/Locals tabs in Visual Studio.*

**Comment by BlyZeDev** [comment]
It plays but doesnt fire TrackStartedEvent and when its finished it doesnt fire TrackEndEvent.
It doesnt fire TrackException or TrackStuckEvent either

**Comment by Yucked** [comment]
What have you tried? Is it only for that song? Does Lavalink fire the event? Does the event show up in Victoria’s log event? Does it throw any exception? Please provide information, I can’t debug your code for you.

**Comment by BlyZeDev** [comment]
I made breakpoints to see if the events get fired but there dont.
Its only for that song, all the other songs from soundcloud, youtube, twitch and youtube music I tried are firing the events as the should.
Lavalink fires the event and it shows up in the log.
It only throws an exception in lavalink. See pictures

Thats when i play the song that doesnt work
![Screenshot 2022-01-07 213914](https://user-images.githubusercontent.com/95288041/148604811-dae59dc2-ace4-47ec-a42a-1b620a580b62.png)

Thats a random other song from soundcloud
![Screenshot 2022-01-07 214053](https://user-images.githubusercontent.com/95288041/148604889-4b367a16-702f-4069-883f-19b152a3cfa1.png)
